src/content/docs/en/guides/cms/keystatic.mdx

```diff
@@ -2,13 +2,263 @@
 
+:::tip
+If you're starting a **new Astro + Keystatic project from scratch**, you can use the [Keystatic CLI](https://keystatic.com/docs/quick-start#keystatic-cli) to generate a new project in seconds:
+
+```bash
+npm create @keystatic@latest
+```
+
+Select the Astro template, and you'll be ready to [deploy](#deploying-keystatic--astro)!
+:::
+
+## Prerequisites
+
+This guide assumes you already have **an existing Astro project** in SSR running on [`output: 'hybrid'`](<>). If you don’t, you can follow the instructions in our [Installation guide](/en/install/auto/).
+
+:::note
+If you intend to sync Keystatic's data with GitHub, you will also need **a GitHub account with `write` permissions** on the repository for this project.
+:::
+
+## Installing dependencies
+
+You're going to need to install a few packages to get Keystatic going.
+
+Keystatic outputs Markdoc and has a dependency on React, so first you need to add Astro's first-party integrations for both:
+
+```bash
+npx astro add react markdoc
+```
+
+You will also need two Keystatic packages:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install @keystatic/core @keystatic/astro
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm install @keystatic/core @keystatic/astro
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add @keystatic/core @keystatic/astro
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+## Creating a Keystatic config file
+
+Keystatic needs a config file. This is where you can connect a project with a specific GitHub repository and define a content schema.
+
+Create a file called `keystatic.config.ts` in the root of the project:
+
+```ts {8-25}
+// keystatic.config.ts
+import { config, fields, collection } from '@keystatic/core';
+
+export default config({
+  storage: {
+    kind: 'local',
+  },
+  collections: {
+    posts: collection({
+      label: 'Posts',
+      slugField: 'title',
+      path: 'src/content/posts/*',
+      format: { contentField: 'content' },
+      schema: {
+        title: fields.slug({ name: { label: 'Title' } }),
+        content: fields.document({
+          label: 'Content',
+          formatting: true,
+          dividers: true,
+          links: true,
+          images: true,
+        }),
+      },
+    }),
+  },
+});
+```
+
+You've exported a config object wrapped in the `config` function imported from `@keystatic/core`.
+
+For now, the `storage` strategy is set to `local`, and there is a single “posts” `collection`.
+
+:::note
+The schema of your Keystatic collections should match the collections you've defined in Astro's [content collections](/en/guides/content-collections/#defining-collections).
+
+We're working on a way to automatically sync Keystatic collections with Astro's content collections... Stay tuned! 🤗
+:::
+
+This is all Keystatic needs to start managing content, configuration-wise.
+
+Now, you need to display the Keystatic Admin UI on our site!
```

Comment:
```suggestion
Keystatic is now configured to manage your content and enforce your schema.
```

Reply:
```suggestion
Keystatic is now configured to manage your content based on your schema.
```

Reply:
I like this! (Even if I had to read it three times at this time of night to figure out how it was different!)  😅  Ship it!